.circleci/config.yml

```diff
@@ -50,9 +50,9 @@ build_and_deploy: &build_and_deploy
     - restore_cache:
         name: Restore Composer Cache
         keys:
-          - v2-dependencies-{{ checksum "composer.lock" }}
+          - v3-dependencies-{{ checksum "composer.lock" }}
```

Comment:
Sorry to do this to you, but as I posted in Slack I found a better solution to the Composer build issue. It involves adding a setting to `composer.json` (see https://github.com/eGovPDX/employees/pull/414/commits/99e2e161529d3897d007c3512cf1924000b0b258) 

So I think it would be better to have you revert your changes to this file and add the new setting to `composer.json`.